Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -211,105 +216,114 @@ Configure the network that will be enabled for the cluster and services.
+Some network configurations allow manually specifying the public and network interfaces that Ceph
+will use for data traffic. Use `addressRanges` to specify this. For example:
 
-If you want to learn more, please read:
-
-* [Ceph Networking reference](https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/).
-* [Multus documentation](https://intel.github.io/multus-cni/doc/how-to-use.html)
-
-Based on the configuration, the operator will do the following:
+```yaml
+  network:
+    provider: host
+    addressRanges:
+      public:
+        - "192.168.100.0/24"
+        - "192.168.101.0/24"
+      cluster:
+        - "192.168.200.0/24"
+```
 
-1. If only the `public` selector is specified, all communication will happen on that network
+Refer to [Ceph networking documentation](https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/)
+for more details.
 
-    ```yaml
-      network:
-        provider: multus
-        selectors:
-          public: rook-ceph/rook-public-nw
-    ```
+#### Host Networking
 
-2. If only the `cluster` selector is specified, the internal cluster traffic\* will happen on that network. All other traffic to mons, OSDs, and other daemons will be on the default network.
+To use host networking, set `provider: host`.
 
-    ```yaml
-      network:
-        provider: multus
-        selectors:
-          cluster: rook-ceph/rook-cluster-nw
-    ```
+To instruct Ceph to operate on specific host interfaces or networks, use `addressRanges` to select
+the network CIDRs Ceph will bind to on the host.
 
-3. If both `public` and `cluster` selectors are specified the first one will run all the communication network and the second the internal cluster network\*
+If the host networking setting is changed in a cluster where mons are already running, the existing mons will
+remain running with the same network settings with which they were created. To complete the conversion
+to or from host networking after you update this setting, you will need to
+[failover the mons](../../Storage-Configuration/Advanced/ceph-mon-health.md#failing-over-a-monitor)
+in order to have mons on the desired network configuration.
 
-    ```yaml
-      network:
-        provider: multus
-        selectors:
-          public: rook-ceph/rook-public-nw
-          cluster: rook-ceph/rook-cluster-nw
-    ```
+#### Multus
 
-\* Internal cluster traffic includes OSD heartbeats, data replication, and data recovery
+Rook supports using Multus NetworkAttachmentDefinitions for Ceph public and cluster networks.
 
-Only OSD pods will have both Public and Cluster networks attached. The rest of the Ceph component pods and CSI pods will only have the Public network attached.
-Rook Ceph operator will not have any networks attached as it proxies the required commands via a sidecar container in the mgr pod.
+Refer to [Multus documentation](https://intel.github.io/multus-cni/doc/how-to-use.html) for details
```

Comment:
The link is not opening for me

Reply:
Looks like this was a really old link that got copied. Should be fixed now.